.github/workflows/update-docker-requirements.yml

```diff
@@ -42,24 +47,40 @@ jobs:
           NEEDS_UPDATE=$(git diff --quiet && echo 'false' || echo 'true')
           echo "needs-update=${NEEDS_UPDATE}" | tee --append $GITHUB_OUTPUT
 
+      - name: Check if commit is associated with a PR
+        id: pr-check
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          PR_NUMBER=$(gh api \
+            -H 'Accept: application/vnd.github+json' \
+            /repos/${{ github.repository }}/commits/${{ github.sha }}/pulls \
+            --jq '.[0].number' || echo "none")
+          if [ "${PR_NUMBER}" != "none" ]; then
+            HAS_PR='true'
```

Comment:
I guess we don't even need this information and can just check the merger or author for "none". Or am I missing something? If need, let's remove it.

Reply:
Removed it, yet to test it. 
When testing this branch on my fork it fails because it cannot find #459 PR on the fork.

Reply:
here is a test run - [https://github.com/arjxn-py/ragna/actions/runs/10183118614/job/28167297481](https://github.com/arjxn-py/ragna/actions/runs/10183118614/job/28167297481)